plugins/modules/aci_access_span_filter_group.py

```diff
@@ -0,0 +1,246 @@
+  sample:
+    [
+        {
+            "fvTenant": {
+                "attributes": {
+                    "descr": "Production",
+                    "dn": "uni/tn-production",
+                    "name": "production",
+                    "nameAlias": "",
+                    "ownerKey": "",
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        filter_group=dict(type="str", aliases=["name"]),  # Not required for querying all objects
+        name_alias=dict(type="str"),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["filter_group"]],
+            ["state", "present", ["filter_group"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    filter_group = module.params.get("filter_group")
+    state = module.params.get("state")
+    name_alias = module.params.get("name_alias")
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="infra",
+            aci_rn="infra",
+        ),
+        subclass_1=dict(
+            aci_class="spanFilterGrp",
+            aci_rn="filtergrp-{0}".format(filter_group),
+            module_object=filter_group,
+            target_filter={"name": filter_group},
+        ),
+        child_classes=["spanFilterEntry"],
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        aci.payload(aci_class="spanFilterGrp", class_config=dict(name=filter_group, nameAlias=name_alias))
+        aci.get_diff(aci_class="spanFilterGrp")
```

Comment:
Leave an empty line after get_diff for consistency with the other modules

Reply:
ok